sympy/polys/tests/test_numberfields.py

```diff
@@ -143,6 +143,12 @@ def test_minimal_polynomial():
     assert minimal_polynomial(sqrt(2)*I + I*(1 + sqrt(2)), x,
             compose=False) ==  x**4 + 18*x**2 + 49
 
+    # minimal polynomial of I
+    assert minimal_polynomial(I, x, domain=QQ.algebraic_field(I)) == x - I
+    K = QQ.algebraic_field(I*(sqrt(2) + 1))
+    assert minimal_polynomial(I, x, domain=K) == x - I
```

Comment:
Both of these tests result in x - I. Is the other possibility, the minimal polynomial being `x**2 + 1` , tested anywhere?

Reply:
I didn't think it was necessary but maybe it could be included. Perhaps this would suffice?
```
>>> minimal_polynomial(I, x, domain=QQ)
x**2 + 1
```
Or this?
```
>>> minimal_polynomial(I, x, domain='QQ(y)')
x**2 + 1
```